frontend/src/features/apartment/ApartmentDetailsPage.tsx

```diff
@@ -81,6 +81,28 @@ const LoadedApartmentDetails = ({data}: {data: IApartmentDetails}): JSX.Element
     const unconfirmedPrices = isPre2011
         ? data.prices.maximum_prices.unconfirmed.pre_2011
         : data.prices.maximum_prices.unconfirmed.onwards_2011;
+
```

Comment:
Maybe it would be good to have this as a service, so it could be reused in other components easily.